packages/docusaurus-mdx-loader/src/remark/toc/index.ts

```diff
@@ -124,9 +178,21 @@ export default plugin;
 async function createExportNode(name: string, object: any): Promise<MdxjsEsm> {
```

Comment:
I know it was like that before, but that object shouldn't be of type `any`. It should have previously been `TOCItem[]`, and now support your additional types